app/helpers/template_helpers.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         self._google_tag_manager_auth = current_app.config.get(
             "EQ_GOOGLE_TAG_MANAGER_AUTH"
         )
+        self._survey_type = cookie_session.get("theme")
```

Comment:
The `cookie_has_theme` in `service_links_context` should be updated to use `self._survey_type` now.